### DISCUSSION TITLE
In a frame, disable autoresize widget or fix the width of a column

### DISCUSSION THREAD

**Discussion by zorrolaotse** [TomSchimansky/CustomTkinter#2335] [Q&A]
HiI,
with CustomCtinker, is there a way to disable the autoresize of a widget in a column of a frame or to fix the width of a column
for example, when a frame contains columns, each column takes the width of its widget
With a frame by row here the result :

aaa.bbb
aaaaa.bbbbb

Here is what I try to produce :

aaa............bbb
aaaaa.......bbbbb

Thank you

**Comment by MaxNiark** [comment]
self.grid_rowconfigure((0,12,3,4), weight=0, minsize=150)
self.grid_columnconfigure((0,12,3,4), weight=0, minsize=150)

**Reply by zorrolaotse** [reply]
Thanks a lot,
combining the weight and minsize is the solution
